app/views/layouts/godmin/_layout.html.erb

```diff
@@ -7,8 +7,9 @@
     <%= csrf_meta_tags %>
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <%= yield :head %>
+    <img src="/logo.jpg" alt="logo" width="128" height="128"/>
```

Comment:
A css class rather than an inline style would be more inline (pun not intended) with current best practice thinking on css styling